src/SimulationCSharpClient/Client/SimulationClientGenerated.cs

```diff
@@ -17115,6 +17131,10 @@ public partial class PartBasedSimulationParameters : System.ComponentModel.INoti
         private System.Collections.ObjectModel.ObservableCollection<double> _distortionAfterCutoffScaleFactorValues;
         private PartBasedSimulationParametersSupportType _supportType;
         private int? _voxelSampleRate;
+        private bool? _includeOnPlateStrainOutput;
+        private bool? _enableContours;
+        private PartBasedSimulationParametersLoadSteppingType? _loadSteppingType;
+        private int? _loadSteps;
```

Comment:
I'm not entirely sure why this change is here. This has nothing to do with my changes to the simulation api spec